lib/path/README.md

```diff
@@ -187,6 +187,27 @@ Decision: All functions remove trailing slashes in their results.
 
 </details>
 
+### Prefer returning subpaths over components
+[subpath-preference]: #prefer-returning-subpaths-over-components
```

Comment:
Here's why this was added: https://github.com/NixOS/nixpkgs/pull/244358#discussion_r1268595145